test.js

```diff
@@ -82,3 +82,12 @@ test('ignore ignore-case for pid', async t => {
 
 	await noopProcessKilled(t, pid);
 });
+
+test('error when process is not found', async t => {
+	try {
+		await m(['notFoundProcess']);
+		t.fail();
+	} catch (err) {
+		t.regex(err.message, /Killing process notFoundProcess failed: Process doesn't exist/);
```

Comment:
You could use t.is here instead since you know the whole error message.

Reply:
Would be better to use `t.throws` here.